code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -660,3 +662,47 @@ Then we space some of our heat, and think about if we should stop conducting.
 			(heat_capacity * sharer.heat_capacity / (heat_capacity + sharer.heat_capacity)) //The larger the combined capacity the less is shared
 		temperature -= heat / heat_capacity //The higher your own heat cap the less heat you get from this arrangement
 		sharer.temperature += heat / sharer.heat_capacity
+
+/**
+ * Adds a source of thermal conductivity to change our conductivity to the one indicated
+ */
+/turf/proc/add_thermal_conductivity_source(conductivity, priority)
+	if(!thermal_conductivities || !LAZYLEN(thermal_conductivities))
+		LAZYSETLEN(thermal_conductivities, MAX_TEMPORARY_THERMAL_CONDUCTIVITY)
+	if(!conductivity)
+		return
+	if(priority > LAZYLEN(thermal_conductivities))
+		return
+	LAZYSET(thermal_conductivities, priority, conductivity)
+	update_thermal_conductivity()
+
+/**
+ * Removes a source of thermal conductivity to change our conductivity to the next one in priority or restore it to the initial value
+ */
+/turf/proc/remove_thermal_conductivity_source(conductivity, priority)
+	if(!thermal_conductivities)
+		return
+	if(priority > LAZYLEN(thermal_conductivities))
+		return
+	if(conductivity && thermal_conductivities[priority] != conductivity)
+		return
+	LAZYSET(thermal_conductivities, priority, null)
+	update_thermal_conductivity()
+
+/**
+ * Updates the thermal conductivity.
+ * Restores it to the initial value, then updates it with the conductivities stored in the list
+ */
+/turf/proc/update_thermal_conductivity()
+	thermal_conductivity = initial(thermal_conductivity)
+	if(!thermal_conductivities)
+		return
+	for(var/checked_conductivity in thermal_conductivities)
+		if(islist(checked_conductivity))
+			var/list/conductivity_list = checked_conductivity
+			if(conductivity_list.len)
+				thermal_conductivity = conductivity_list
+				return
+		else if(checked_conductivity)
+			thermal_conductivity = checked_conductivity
+			return
```

Comment:
As it is, if a turf gains thermal conductivity it'll never lose it (damn you atom color code)

Do you mind putting a `LAZYNULL` here at the end so it returns to being a proper lazylist?